### PR TITLE
Minor cmake updates to make the build easier

### PR DESCRIPTION
**Pull request by p0wdrdotcom**
Added resolution of ACML path to cmake build
Added requirement flag on cuda to cmake build
Added install target to cmake build
Added cuda linking to cmake build
Added readme as a result of my build process

Tested on Ubuntu 16.04 with ACML 5.3.1, CUDA 8.0 and a Geforce GTX 970

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+ * Install cuda 8.0
+ * Install gcc-5
+ * Install ACML
+ * Run cmake and build libdamascene.so
+ * Build the python bindings
+ * ... start GPU segmenting
+
+### Install CUDA
+
+This assumes you have installed the proprietary nvidia drivers for ubuntu.  (although I think the cuda intall process may take care of that)
```
Comment (amueller) [inline README.md]: *install ;)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+```
+wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64/cuda-repo-ubuntu1604_8.0.61-1_amd64.deb
```
Comment (amueller) [inline README.md]: Is that 16.04 specific?

Comment (p0wdrdotcom) [reply README.md]: Yes it is.
There is a 14.04 version available too.
https://developer.nvidia.com/cuda-downloads

Comment (amueller) [reply README.md]: Can you maybe make a note of that?

**Comment by amueller** [comment]
thanks :)

**Pull request merged**